recipients_app/lib/ui/inputs/input_text.dart

```diff
@@ -37,6 +39,7 @@ class InputText extends StatelessWidget {
             // labelText: hintText,
             suffixIcon: suffixIcon,
             floatingLabelBehavior: FloatingLabelBehavior.never,
+            counter: const SizedBox.shrink()
```

Comment:
What is this good for and why is it empty?

Reply:
When the characters limit is added Flutter is adding the counter on the bottom of the text input automatically. Which is not what we want AFAIK. One of the solutions (which also make it not adding extra space to the bottom of input fields is like above.

Reply:
Source https://stackoverflow.com/a/58819500

Reply:
maybe add this as a comment, otherwise we will stumble over this again :D?

Reply:
sure 👍